recipes/pango/all/conanfile.py

```diff
@@ -0,0 +1,140 @@
+import os
+import shutil
+import glob
+
+from conans import ConanFile, tools, Meson, VisualStudioBuildEnvironment
+
+
+class PangoConan(ConanFile):
+    name = "pango"
+    license = "LGPL-2.0-and-later"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Internationalized text layout and rendering library"
+    homepage = "https://www.pango.org/"
+    topics = ("conan", "fontconfig", "fonts", "freedesktop")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+    generators = "pkg_config"
+    _source_subfolder = "source_subfolder"
+    _autotools = None
+
+    @property
+    def _is_msvc(self):
+        return self.settings.compiler == "Visual Studio"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.libcxx
+        del self.settings.compiler.cppstd
+
+    def build_requirements(self):
+        if not tools.which("pkg-config"):
+            self.build_requires("pkgconf/1.7.3")
```

Comment:
```suggestion
        self.build_requires("pkgconf/1.7.3")
```

Force the specific version. Why? Any different behavior which cause an error and someone reports as an issue, we won't be able to reproduce the same scenario. Okay, Cmake is an exception.